js/committer-cards/committer-list.json

```diff
@@ -0,0 +1,162 @@
+[
+    {
+        "handle": "aguibert",
+        "name": "Andrew Guibert",
+        "website": ""
```

Comment:
@edbratt I liked your idea using website Github field (for some reason it's name "blog" field in GH API) but as I've checked some people have broken links there, so GH profile might be a less faulty approach. I left values in json, but not used in the template, for now, as it might need further discussion, you can still reach them via GH profile link.

Reply:
gotcha Andrii.

Can you adjust the title of this ticket to the Contributor Card instead? 

We the scope covers Committer & Contributors alike. Again, thanks for pushing awesomeness to this task : )